src/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -284,13 +283,12 @@ public byte[] ToByteArray()
             byte highByte;
             int nonZeroDwordIndex = 0;
             uint highDword;
-            uint[] bits = _bits;
-            if (bits == null)
+            if (_bits == null)
             {
-                highByte = (byte)((sign < 0) ? 0xff : 0x00);
-                highDword = unchecked((uint)sign);
+                highByte = (byte)((_sign < 0) ? 0xff : 0x00);
+                highDword = unchecked((uint)_sign);
```

Comment:
Nope. I'm unable to ngen since the strong name validation fails. How can I validate that?